enterprise/cmd/frontend/internal/authz/init.go

```diff
@@ -92,6 +88,11 @@ func Init(ctx context.Context, db database.DB, _ conftypes.UnifiedWatchable, ent
 				authzNames = append(authzNames, t)
 			}
 		}
+
+		if len(seriousProblems) == 0 {
+			return nil
+		}
```

Comment:
```suggestion
		if len(authzNames) == 0 {
			return nil
		}
```

I think it makes more sense to check the length of the `authzNames` here because we are not using the `seriousProblems` any further but only `authzNames`.